### ISSUE TITLE
Yaml file order is not maintained

### ISSUE THREAD

**Issue by mikefarah** [mikefarah/yaml#5]
When using the write command, the order of the fields in the Yaml file is not maintained.

```
version: '2'
services:
  test:
    image: ubuntu:14.04
    stdin_open: true
    tty: true
```
yaml w docker-compose.yml services.test.image nginx:master

```
services:
  test:
    image: nginx:master
    stdin_open: true
    tty: true
version: "2"
```

**Comment by mikefarah** [comment]
Originally raised in [Incorrect parsing of file with nested key](https://github.com/mikefarah/yaml/issues/4)

**Comment by mikefarah** [comment]
Note: Probably need to use [MapSlice](https://godoc.org/gopkg.in/yaml.v2#MapSlice) to maintain the order

**Comment by mikefarah** [comment]
Fixed in Release 1.6 @jmreicha
